doc/manual.md

```diff
@@ -69,6 +69,12 @@ For communication with the ReShare back-end:
 * `username` -- The name of the user who resource-sharing requests should be posted as. This should be a user who has all the necessary permissions, but it is preferable to avoid using an administrator -- just as one does not usually run Unix services as root. A special user such as `openurl` might be created for this purpose.
 * `password` -- The password needed to authenticate the specified user.
 
+For preferentially sourcing the patron id from a header (eg. provided by SSO):
+
+* `reqIdHeader` -- The HTTP header providing the patron id. Typically this will be explicitly populated by a proxy that will strip any pre-existing value.
+* `reqIdToLower`, `reqIdToUpper` -- Boolean options that transform the case of the incoming id from a header.
+* `reqIdRegex`, `reqIdReplacement` -- Strings to facilitate transform of the incoming id from a header, syntax following the the [Javascript replace function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace) that implements them. Example: `{ reqIdRegex: 'abc-(.*)', reqIdReplacement: '$1-b' }`
+
```

Comment:
I like how simple the configuration is here.

What do you mean by "preferentially"?

Reply:
I mean that at the moment it'll prefer the value from the header if present but still use the value from the parameters otherwise. That's just what I went with to start however, we may reasonably decide that we want it to fail if a header is designated for a symbol and isn't present.

Reply:
Oh, I see. It's worth making explicit that this is there to override `req_id`.